autogluon/utils/tabular/ml/models/ensemble/bagged_ensemble_model.py

```diff
@@ -254,6 +258,9 @@ def compress(self, X, y):
         model_compressed.fit(X_train=X, Y_train=y)  # TODO: This only works for stacker, not for bagged
         return model_compressed
 
+    def convert_to_compressed_template(self):
```

Comment:
I believe this can be removed

Reply:
I added it, because in line 253 (on master branch) we have this: 
` model_compressed = self.convert_to_compressed_template()`, but there is no implementation of `convert_to_compressed_template`. Is this mean that `compress` is also not needed?

Reply:
Oh interesting, this might be a mistake from the renaming of the compression function to refit_full. I'll take a look.

Reply:
I see it now, thanks! I'll likely send a PR to remove it.